sdks/python/apache_beam/io/fileio.py

```diff
@@ -220,22 +235,33 @@ def close(self, file_handle):
     if file_handle is not None:
       file_handle.close()
 
+  @check_accessible(['file_path_prefix', 'file_name_suffix'])
```

Comment:
Java SDK does not have a concept similar to `check_accesible` decorators. The common API across both languages are `is_accesible` and `get`, where the second one will throw a `RuntimeException` if the underlying value is not accesible (i.e. called at pipeline building time).

Using `check_accessible` is optional. A PTransform author can build a templatable transform without using it. Another pro is, it improves the readability by clearly marking functions that depend on run time values of some value.